angular-material.scss

```diff
@@ -0,0 +1,424 @@
+  --md-sys-typescale-headline2-line-height: 40px;
+  --md-sys-typescale-headline2-tracking: 0px;
+
+  --md-sys-typescale-subhead1-font: Roboto;
+  --md-sys-typescale-subhead1-weight: Medium;
+  --md-sys-typescale-subhead1-size: 16px;
+  --md-sys-typescale-subhead1-line-height: 24px;
+  --md-sys-typescale-subhead1-tracking: 0.1px;
+
+  --md-sys-typescale-subhead2-font: Roboto;
+  --md-sys-typescale-subhead2-weight: Medium;
+  --md-sys-typescale-subhead2-size: 14px;
+  --md-sys-typescale-subhead2-line-height: 20px;
+  --md-sys-typescale-subhead2-tracking: 0.1px;
+
+  --md-sys-typescale-body1-font: Roboto;
+  --md-sys-typescale-body1-weight: Regular;
+  --md-sys-typescale-body1-size: 16px;
+  --md-sys-typescale-body1-line-height: 24px;
+  --md-sys-typescale-body1-tracking: 0.5px;
+
+  --md-sys-typescale-body2-font: Roboto;
+  --md-sys-typescale-body2-weight: Regular;
+  --md-sys-typescale-body2-size: 14px;
+  --md-sys-typescale-body2-line-height: 20px;
+  --md-sys-typescale-body2-tracking: 0.25px;
+
+  --md-sys-typescale-caption-font: Roboto;
+  --md-sys-typescale-caption-weight: Regular;
+  --md-sys-typescale-caption-size: 32px;
+  --md-sys-typescale-caption-line-height: 16px;
+  --md-sys-typescale-caption-tracking: 0.4px;
+
+  --md-sys-typescale-button-font: Roboto;
+  --md-sys-typescale-button-weight: Medium;
+  --md-sys-typescale-button-size: 14px;
+  --md-sys-typescale-button-line-height: 20px;
+  --md-sys-typescale-button-tracking: 0.1px;
+
+  --md-sys-color-primary-light: #6750A4;
+  --md-sys-color-on-primary-light: #FFFFFF;
+
+  --md-sys-color-secondary-light: #625B71;
+  --md-sys-color-on-secondary-light: #FFFFFF;
+
+  --md-sys-color-error-light: #B3261E;
+  --md-sys-color-on-error-light: #FFFFFF;
+
+  --md-sys-color-primary-dark: #D0BCFF;
+  --md-sys-color-on-primary-dark: #381E72;
+
+  --md-sys-color-secondary-dark: #CCC2DC;
+  --md-sys-color-on-secondary-dark: #332D41;
+
+  --md-sys-color-error-dark: #F2B8B5;
+  --md-sys-color-on-error-dark: #601410;
+}
+
+// Define the palettes for your theme using the Material Design palettes available in palette.scss
+// (imported above). For each palette, you can optionally specify a default, lighter, and darker
+// hue. Available color palettes: https://material.io/design/color/
+
+// Define your Angular theme with color palettes, typography and density
+$theme-primary-palette: (
+  50: var(--md-sys-color-primary-light),
+  100: var(--md-sys-color-primary-light),
+  200: var(--md-sys-color-primary-light),
+  300: var(--md-sys-color-primary-light),
+  400: var(--md-sys-color-primary-light),
+  500: var(--md-sys-color-primary-light),
+  600: var(--md-sys-color-primary-light),
+  700: var(--md-sys-color-primary-light),
+  800: var(--md-sys-color-primary-light),
+  900: var(--md-sys-color-primary-light),
+  A100: var(--md-sys-color-primary-light),
+  A200: var(--md-sys-color-primary-light),
+  A400: var(--md-sys-color-primary-light),
+  A700: var(--md-sys-color-primary-light),
```

Comment:
@twerske I am wondering if it is ok to use the same color for different tones and shouldn't we use something like this?
```scss
        50: var(--md-ref-palette-primary95),
        100: var(--md-ref-palette-primary90),
        200: var(--md-ref-palette-primary80),
        300: var(--md-ref-palette-primary70),
        400: var(--md-ref-palette-primary60),
        500: var(--md-ref-palette-primary40),
        600: var(--md-ref-palette-primary40),
        700: var(--md-ref-palette-primary30),
        800: var(--md-ref-palette-primary20),
        900: var(--md-ref-palette-primary10),
```